src/Symfony/Component/Ldap/composer.json

```diff
@@ -22,11 +22,13 @@
         "ext-ldap": "*"
     },
     "require-dev": {
-        "symfony/security-core": "^5.3"
+        "symfony/security-core": "^5.3",
+        "symfony/security-http": "^5.2"
     },
     "conflict": {
         "symfony/options-resolver": "<4.4",
-        "symfony/security-core": "<5.3"
+        "symfony/security-core": "<5.3",
+        "symfony/security-http": "<5.2"
```

Comment:
We can discuss removing the conflict rule if it's too heavy. My analysis:

* `security-http` 4.4 cannot be installed anyway because if requires `security-core` 4.4.
* `security-http` 5.0 could have been installed previously, but it's really dead and buried.
* `security-http` 5.1 is incompatible. The component is already assuming the 5.2 version of the new authenticator system.